### ISSUE TITLE
pl-order-blocks dragging may be difficult with page scrolling

### ISSUE THREAD

**Issue by echuber2** [PrairieLearn/PrairieLearn#5748]
If a page with pl-order-blocks won't fit in one screen, the user may be need to scroll while dragging blocks. This is especially concerning if the solution building area is placed below the block tray instead of on the right-hand side. (Code organization problems have long line widths, so placing the solution area below would be a typical use case.)

We probably need a way to simplify the dragging if scrolling would be needed. For example, perhaps as an alternative to dragging, you should be able to discretely select a block to move and a slot to move it into. This may also go along with #4912 (keyboard accessibility).

**Comment by ffund** [comment]
Related problem that could also be addressed by a solution to this issue: sometimes, even if you scroll all the way down, a large block cannot be placed in the solution area unless you happened to click-to-drag near the top of the block.

For example, in this screenshot I have scrolled all the way down (can't scroll any more), then clicked near the bottom of this long block and dragged it down as far as possible (can't move cursor any lower), but the block will still be put in the source blocks section. 

![screenshot-example](https://user-images.githubusercontent.com/2316553/166457468-2d9052f8-7010-49d2-87c1-6f953a963704.png)

(To put this block in a solution, I have to click-to-drag near the top/middle of the block - then I'm able to drag it down far enough.)

**Comment by echuber2** [comment]
That's a good example. I think the operating system's window manager controls whether the mouse cursor can be dragged off-screen during an operation like this, so there may be no easy workaround. Perhaps the block should be represented as an abbreviated chunk of a fixed size during the drag operation so that it is easy to manipulate on-screen too. Worth making a separate issue?

**Comment by echuber2** [comment]
(Alternatively, question authors may need to redact enough content in a block so that each block is of a reasonable size.)
